src/gui/sharedialog.cpp

```diff
@@ -152,48 +152,49 @@ ShareDialog::ShareDialog(QPointer<AccountState> accountState,
     }
 }
 
-void ShareDialog::addLinkShareWidget(const QSharedPointer<LinkShare> &linkShare){
+ShareLinkWidget *ShareDialog::addLinkShareWidget(const QSharedPointer<LinkShare> &linkShare){
```

Comment:
since we're changing the line anyway, it's a good opportunity to break the line before {

Reply:
@er-vin Oh, a break before `ShareLinkWidget *ShareDialog::addLinkShareWidget(const QSharedPointer<LinkShare> &linkShare){`?

Well, I've checked elsewhere and couldn't find any function with a break before the first operator. But, I can add spaces before `{` instead as they are missing here.

Reply:
Well most files have that style:
```cpp
void Class::method()
{
```

Just turns out that this file has a different style somehow. That's why I mentioned fixing that line here to slowly realign this with the rest of the product. Since there's action going on in the share dialog lately and if done consistently, it'll slowly get back to the style of the other files.

Reply:
@er-vin aaaaaah, yes, I've been thinking about a completely different thing